### PR TITLE
Timezonefinder Recipe

### PR DESCRIPTION
**Pull request by snowman2**

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,72 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml]: Please do not delete the example recipe

File: recipes/timezonefinder/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - numba    # [py>27]
```
Comment (synapticarbors) [inline recipes/timezonefinder/meta.yaml]: Why is numba only for `>py27`? Numba is supported for py2 and 3 and I don't see any indication in the source repo's README that the library doesn't support numba for py27.

Comment (synapticarbors) [inline recipes/timezonefinder/meta.yaml]: ok, just saw the issue on the source repo's issue tracker.

File: recipes/example/meta.yaml (hunk 1 above)
Comment (snowman2) [inline recipes/example/meta.yaml]: Whoops. It is fixed now.

**Comment by synapticarbors** [comment]
@snowman2 I fixed the python 2.7 issue upstream, so next time there is a release of this library, please make sure to update the recipe.

**Comment by snowman2** [comment]
Will do. Thanks!

**Pull request merged**